common/shlibs

```diff
@@ -49,6 +49,35 @@ libcidn.so.1 glibc-2.32_1
 libmvec.so.1 glibc-2.32_1
 libcddgmp.so.0 cddlib-0.94m_1
 libcdd.so.0 cddlib-0.94m_1
+libc.so.6 glibc-2.34_1
+libm.so.6 glibc-2.34_1
+libpthread.so.0 glibc-2.34_1
+librt.so.1 glibc-2.34_1
+libdl.so.2 glibc-2.34_1
+ld-linux-x86-64.so.2 glibc-2.34_1 x86_64
+ld-linux.so.2 glibc-2.34_1 i686
+ld-linux.so.3 glibc-2.34_1 armv5tel
+ld-linux-aarch64.so.1 glibc-2.34_1 aarch64
+ld64.so.2 glibc-2.34_1 ppc64
+ld.so.1 glibc-2.34_1 mips
+ld.so.1 glibc-2.34_1 ppc
+ld-linux-armhf.so.3 glibc-2.34_1
+libresolv.so.2 glibc-2.34_1
+libanl.so.1 glibc-2.34_1
+libthread_db.so.1 glibc-2.34_1
+libutil.so.1 glibc-2.34_1
+libnsl.so.1 glibc-2.34_1
+libnss_db.so.2 glibc-2.34_1
+libnss_files.so.2 glibc-2.34_1
+libnss_compat.so.2 glibc-2.34_1
+libnss_dns.so.2 glibc-2.34_1
+libnss_hesiod.so.2 glibc-2.34_1
+libcrypt.so.1 glibc-2.34_1
+libBrokenLocale.so.1 glibc-2.34_1
+libmemusage.so glibc-2.34_1
+libSegFault.so glibc-2.34_1
+libpcprofile.so glibc-2.34_1
+libmvec.so.1 glibc-2.34_1
```

Comment:
If you're updating to 2.35, do these not need to be adjusted as well? i.e. not point to 2.34